test/server/guarddog_impl_test.cc

```diff
@@ -208,17 +208,15 @@ TEST(WatchDogBasicTest, ThreadIdTest) {
   gd.stopWatching(watched_dog);
 }
 
-// If this test fails it is because the SystemTime object has become nontrivial
-// or we are compiling under a compiler and library combo that makes the
-// SystemTime object require a lock to be atomicly modified.
+// If this test fails it is because the std::time_t type has become nontrivial or we are compiling
```

Comment:
Comment nit: std::time_t is generally the legacy "seconds since epoch" but is actually ill-defined in the C++ spec.  Implied is that it is only second granularity like the old C type so not suitable for this application.